composables/transaction/transactionConsume.ts

```diff
@@ -0,0 +1,24 @@
+import { Interaction, createInteraction } from '@kodadot1/minimark'
+
+import { bsxParamResolver, getApiCall } from '@/utils/gallery/abstractCalls'
+import type { ActionConsume } from './types'
+
+export function exectConsumeTx(item: ActionConsume, api, executeTransaction) {
```

Comment:
```suggestion
export function execBurnTx(item: ActionConsume, api, executeTransaction) {
```

Reply:
should I change `transactionConsume` to `tansactionBurn` too?

Reply:
> I was pretty sure you wouldn't like this name hehe

yeah, typos are tricky sometimes

> should I change transactionConsume to tansactionBurn too?

Yes please, it will be burn soon

Reply:
https://github.com/kodadot/packages/pull/91